### PR TITLE
Added mkdir command implementation in terminal

### PR DESCRIPTION
**Pull request by rohan-naik07**
Hey, I am Rohan from Pune. I am interested in your codebase and like to contribute to improve the project. As a start I have implemented "mkdir" ubuntu command in your terminal implementation. On running command "mkdir {folder-name}" a new folder gets added in the desktop.
Please go through the implementation, suggest changes if any, else I would be happy if you merge this in your main codebase :)

### CODE REVIEW

File: src/components/apps/terminal.js [hunk 1]
```diff
@@ -228,6 +228,10 @@ export class Terminal extends Component {
                     result = `ls: cannot access '${words}': No such file or directory                    `;
                 }
                 break;
+            case "mkdir" : 
+                this.props.addFolder(words[0]); 
+                result = `Successfully added ${words[0]} folder in desktop`;
```
Comment (vivek9patel) [inline src/components/apps/terminal.js:233]: here, check if words[0] is null or empty, other wise website will crash!

Comment (vivek9patel) [reply src/components/apps/terminal.js:233]: also, add this command to the list of available commands at line 254!

Comment (vivek9patel) [inline src/components/apps/terminal.js:233]: put result = "", as it is in original terminal

File: src/components/screen/desktop.js [hunk 2]
```diff
@@ -419,28 +420,32 @@ export class Desktop extends Component {
         this.setState({ showNameBar: true });
     }
 
+    addFolder = (folder_name) => {
```
Comment (vivek9patel) [inline src/components/screen/desktop.js:423]: change this function name to "addToDesktop", for future confusion!

Comment (rohan-naik07) [reply src/components/screen/desktop.js:423]: Okay .... I will do the nesessary changes and commit

**Comment by vivek9patel** [comment]
LGTM!

**Comment by vivek9patel** [comment]
can you do similar thing with `code .` command
to open vscode?

**Comment by rohan-naik07** [comment]
Yeah would definitely give it a try !

**Pull request merged**